requirements/test.txt

```diff
@@ -10,7 +10,7 @@ isort==4.2.5
 lxml==3.6.1
 mock==2.0.0
 pep8==1.7.0
-pytest==3.1.0
+pytest==4.0.0
```

Comment:
why we need to update pytest?

Reply:
Our dependent library is calling the function from the newer version pytest. `iter_markers` was deprecated in 4.0.0 although we were using the right function in our dependent library `get_closet_marker` but we were still using old version3.1.0.